### PR TITLE
Yo check these bad ass links out

### PR DESCRIPTION
**Pull request by hidesminimally**
TODO
[] Link Text styles @gracesnoh can u help me with this 
[] Slug animation on the side :>

### CODE REVIEW

File: reflex-app/src/scenes/Home/Home.js [hunk 1]
```diff
@@ -56,9 +65,28 @@ const CTA = styled.button`
...
+
+const ContentContainer = styled.div`
+  display: grid; 
+  grid-template-columns: 1fr 4fr;
+  grid-template-areas: "m c c c" "m c c c" "m c c c";
+  grid-gap: 15px 20px;
+`
+
+const TableOfContents = styled.div`
+  height: 100vh; // ???
```
Comment (hidesminimally) [inline reflex-app/src/scenes/Home/Home.js]: @gracesnoh IDK what I'm doing here

Comment (hidesminimally) [reply reflex-app/src/scenes/Home/Home.js]: image

Comment (gracesnoh) [reply reflex-app/src/scenes/Home/Home.js]: @lilshim I refactored some stuff and moved the `.selected` style into styled components. The link change behavior seems a little slow to me though, I'll be viewing an animation on the right but it won't reflect the correct one on the left.
Screen Shot 2019-04-14 at 1 40 11 PM

Comment (hidesminimally) [reply reflex-app/src/scenes/Home/Home.js]: Yea I notice that too... I'll take a stab at fixing

**Pull request merged**